__tests__/index.test.js

```diff
@@ -236,4 +236,39 @@ describe("my Express app", () => {
         });
     });
   });
+  describe("GET /api/reviews/:review_id/comments", () => {
+    test("200: responds with comments for given review-id", () => {
+      return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then(({ body: { reviews } }) => {
+          reviews.forEach((review) => {
```

Comment:
You're response is an array of comments, not reviews. Change the key you're using here but the properties look sound. 

You don't check how many comments come back and if they belong the the requested review. Make your assertions more robust as sending back an array of every comment will pass this test.